crispy_forms/layout.py

```diff
@@ -139,24 +139,54 @@ class ButtonHolder(LayoutObject):
     """
     Layout object. It wraps fields in a <div class="buttonHolder">
 
-    This is where you should put Layout objects that render to form buttons like Submit.
-    It should only hold `HTML` and `BaseInput` inherited objects.
+    This is where you should put Layout objects that render to form buttons
+    like Submit. It should only hold `HTML` and `BaseInput` inherited objects.
+
+    Attributes:
+        template : str
```

Comment:
Not sure if we should duplicate this as an attribute to the class and as an arg to __init__. Maybe just include in the `args?`.